### DISCUSSION TITLE
Import

### DISCUSSION THREAD

**Discussion by VictorBagz** [community/community#147841] [Repositories] [answered]
How can I import a code folder into my github

**Comment by janithjay** [comment]
1. Click the + icon in the top-right corner and select New repository.
2. Provide a name for your repository, add a description (optional), and choose whether it will be public or private.
3. Click Create repository.
4. Open a terminal or command prompt.
5. Navigate to the folder containing your code
   ```bash
   cd /path/to/your/code-folder
   ```
6. Initialize a Git repository
   ```bash
   git remote add origin https://github.com/your-username/your-repo-name.git
   ```
7. Add all files in the folder to the staging area
   ```bash
   git add .
   ```
8. Commit the changes
   ```bash
   git commit -m "Initial commit"
   ```
9. Push the code to the main branch
   ```bash
   git branch -M main
   git push -u origin main
   ```
10. Go to your repository on GitHub to confirm the files are uploaded.

**Reply by VictorBagz** [reply]
Thank you so much

**Reply by queenofcorgis** [reply, marked as answer]
@VictorBagz thanks for asking this question, if @janithjay solved your question - don't forget to click the `Mark as answer` button ✅

**Comment by VIPSHAKH** [comment]
Simply, Use github desktop
